openedx/core/lib/json_utils.py

```diff
@@ -0,0 +1,29 @@
+"""
+Helpers for json serialization
+"""
+
+import datetime
+from django.core.serializers.json import DjangoJSONEncoder
+from opaque_keys.edx.keys import CourseKey, UsageKey
+
+
+class EdxJSONEncoder(DjangoJSONEncoder):
+    """
+    Custom JSONEncoder that handles `Location` and `datetime.datetime` objects.
+
+    `Location`s are encoded as their url string form, and `datetime`s as
+    ISO date strings
```

Comment:
Don't we want to handle Proxy objects here too?

Reply:
I have changed it the class to inherit **`DjangoJSONEncoder`** instead of **`json.JSONEncoder`**

The django encoder already handles lazy text perfectly. No need for explicitly adding it here

Reply:
Oh, I missed that! My mistake. :) Sorry.

Reply:
No sorry,,, Thank you for your reviews!